app/controllers/event_registrations_controller.rb

```diff
@@ -0,0 +1,144 @@
+# encoding:UTF-8
+class EventRegistrationsController < ApplicationController
+  
+    before_action :login_required
+		before_action :is_registrable, only: [:new, :create]
+		before_action :authenticate_admin, only: :event_index
+    before_action :authenticate, only: [:edit,:update,:show,:destroy]
+		before_action :authenticate_user, only: :profile_index
+    before_action :set_event_registration, only: [:show,:edit,:update,:destroy]
+    before_action :set_event, only: [:destroy, :new,:create, :edit, :index, :update]
+
+	def index
```

Comment:
Assignment Branch Condition size for index is too high. [24.86/15]<br>Inconsistent indentation detected.<br>Tab detected.